filtered-names/filteredNames.js

```diff
@@ -1,6 +1,9 @@
+const fs = require('fs');
 
-function filteredNames(...args) {
-  return args;
+function filteredNames(dirPath, fileType) {
+  const files = fs.readdirSync(dirPath);
+  const filteredFileNames = files.filter(file => file.endsWith(`.${fileType}`));
```

Comment:
won't it work if we simply pass the filetype in the filter condition?
Like so: file.endsWith(fileType)